bigquery/indexer.py

```diff
@@ -317,19 +317,22 @@ def create_samples_json_export_file(es, index_name, deploy_project_id):
             })
 
     client = storage.Client(project=deploy_project_id)
+    user = client.get_service_account_email(
+        project=deploy_project_id).split('@')[0]
```

Comment:
This is the service account associated with the deploy project which means if me you and Melissa all index 1000 genomes it will always be the same file. I think we want to use the username from application default credentials here instead (which will be the service account when deployed to GAE).

Reply:
+1. Looks like you [can't get username from Python client library](https://github.com/googleapis/google-cloud-python/issues/1934#issuecomment-427939993). How about call the shell command "gcloud auth list"?

For local runs, "gcloud auth list" gives your personal email. On App Engine, this should give App Engine service account.

Please make sure this works for both local runs and App Engine. (For App Engine, don't use test-data-explorer; use your own test project.)